README.md

```diff
@@ -26,3 +26,37 @@ import { Pie } from "nanoplot/pie";
 </Graph>
 ```
 
+Step 2: Clone your Forked Repository
```

Comment:
Move this into CONTRIBUTING.md please a file you can create.

Delete anything not in master and not to do with contributing i.e delete this.
```
### Worldmap

- Make sure countries -> paths on map are mapped correctly.
- Make sure PathUtils.center is more accurate. (gpt generated and inaccurate)
- Use flag images for every country, and find CDN that's not what it's set to currently.
- Potential gradient helpers.
- Ensure tooltips have progressive enhancement from server->client, by adding ability to position tooltip to mouse via client only code.

### LineGraph

- Ensure tooltips have progressive enhancement from server->client
- 
```

It was from when i was tracking my ideas in README and is no longer relevant.